apps/condo/pages/tls.tsx

```diff
@@ -0,0 +1,404 @@
+import { LOCALES } from '@condo/domains/common/constants/locale'
+import { PosterWrapper } from '@condo/domains/user/components/containers/styles'
+
+const LOGO_HEADER_STYLES = { width: '100%', justifyContent: 'space-between' }
+const HEADER_LOGO_STYLE: React.CSSProperties = { cursor: 'pointer' }
+const TEXT_CENTER_STYLE: React.CSSProperties = { textAlign: 'center' }
+
+const { publicRuntimeConfig: { defaultLocale } } = getConfig()
+
+const GUIDES_CONTENT = {
+    ru: {
+        windows: ruWindows,
+        macos: ruMacos,
+        linux: ruLinux,
+        android: ruAndroid,
+        ios: ruIos,
+    },
+    en: {
+        windows: enWindows,
+        macos: enMacos,
+        linux: enLinux,
+        android: enAndroid,
+        ios: enIos,
+    },
+}
+
+export const Header: React.FC = (): JSX.Element => {
+    const intl = useIntl()
+    const BackMessage = intl.formatMessage( { id: 'pages.condo.tls.back' })
+
+    const { push } = useRouter()
+    const { isAuthenticated } = useAuth()
+
+    const handleLogoClick = useCallback(() => {
+        if (isAuthenticated) {
+            push('/')
+        } else {
+            push('/auth/signin')
+        }
+    }, [isAuthenticated, push])
+
+    return (
+        <Row style={LOGO_HEADER_STYLES}>
+            <Col style={HEADER_LOGO_STYLE}>
+                <Logo onClick={handleLogoClick}/>
+            </Col>
+            <Col>
+                <Typography.Text>
+                    <Link href='/auth/signin'>
+                        <Typography.Link>{BackMessage}</Typography.Link>
+                    </Link>
+                </Typography.Text>
+            </Col>
+        </Row>
+    )
+}
+
+/**
+ * Does not adds extra elements around anchor component
+ */
+const BlankAnchor = styled(Anchor)`
+    .ant-anchor-link {
+        padding: 0;
+    }
+`
+
+type PosterFooterCardProps = {
+    logoSrc: string
+    description: string
+    // If value is an anchor, like "#guide", then Button on click should scroll to appropriate anchor
+    href: string
+    buttonLabel: string
+}
+
+const PosterFooterCard: React.FC<PosterFooterCardProps> = ({ logoSrc, description, href, buttonLabel }): JSX.Element => (
+    <Card>
+        <Space size={24} direction='horizontal'>
+            <Image src={logoSrc} preview={false}/>
+            <Space size={8} direction='vertical'>
+                <Typography.Paragraph>
+                    {description}
+                </Typography.Paragraph>
+                {href[0] === '#' ? (
+                    <BlankAnchor affix={false}>
+                        <Anchor.Link
+                            href={href}
+                            title={<Button type='secondary' href={href}>{buttonLabel}</Button>}
+                        />
+                    </BlankAnchor>
+                ) : (
+                    <Button type='secondary' href={href}>{buttonLabel}</Button>
+                )}
+            </Space>
+        </Space>
+    </Card>
+)
+
+const MoreLinkStyle = {
```

Comment:
Why link styles differ from our kit?

Reply:
The page by design has some non-standard parts. Designers wanted to look like this

Reply:
Did you asked them why? Because design system is not a set of components. It's a set of agreements which are created to keep code consistent, keep design consistent and be in sync with each other.

I agree, that sometimes you need to create something unique because current set of components can't solve your problems (like anchors and collapses in this PR), but don't think that a title or link is a part of this exception

Reply:
Yes, they just wanted to look like this, because it is a design solution for specific one of a kind case.